### PR TITLE
Update greedy-best-first-search.md

### LINKED ISSUE #3565: [Edit] Subject: Greedy Best First Search Algorithm
### Type of Edit (select all that apply)

Clarify unclear content and/or terms

### Description (optional)

Greedy Best First Search is not properly explained.

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [X ] Issue is assigned

### PR DESCRIPTION
**Pull request by Sahilkumar19**
Fixes #3565  
This PR improving the content of Greedy Best First algorithm.I have added point which were missing or unclear in the provided explanation.

### CODE REVIEW

File: content/ai/concepts/search-algorithms/terms/greedy-best-first-search/greedy-best-first-search.md
Comment (SSwiniarski) [inline content/ai/concepts/search-algorithms/terms/greedy-best-first-search/greedy-best-first-search.md:15]:
```suggestion
**Greedy best-first search**  is an informed search algorithm where the evaluation function is strictly equal to the heuristic function, disregarding the edge weights in a weighted graph because it only cares about the heuristic value. In order to search for a goal node it expands the node that is closest to the goal as determined by the heuristic function. This assumes that it is likely to lead to a solution quickly. However, the solution from Greedy best-first search may not be optimal since a shorter path may exist. 

In this algorithm the search cost is at a minimum since the solution is found without expanding a node that is not on the solution path. This algorithm is minimal, but not complete, since it can lead to a dead end. It's called "Greedy" because at each step it tries to get as close to the goal as it can.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Sahilkumar19 I did a first review and had one suggestion revising your edits for clarity.

**Comment by Sahilkumar19** [comment]
> @Sahilkumar19 I did a first review and had one suggestion revising your edits for clarity. You also need to sigh the [CLA](https://cla-assistant.io/Codecademy/docs?pullRequest=3568).

Thanks for reviewing and suggestions.

**Review by SSwiniarski — APPROVED** [review]
@Sahilkumar19 I made one typo edit and looks good for a second review.

**Comment by SSwiniarski** [comment]
@Sahilkumar19 I did my review. The second review will be by one of the other maintainers.

**Review by caupolicandiaz — APPROVED** [review]
@Sahilkumar19 @SSwiniarski I just made some direct edits to clean up a couple of typos and adjust the phrasing a bit. LGTM

**Pull request merged**
